### PR TITLE
fix hadoop dependency with rubix-presto

### PR DESCRIPTION
**Pull request by pvam**
This fixes Rubix master UT.

### CODE REVIEW

File: rubix-presto/pom.xml [hunk 1]
```diff
@@ -21,9 +21,8 @@
         </dependency>
 
         <dependency>
-            <groupId>org.apache.hadoop</groupId>
```
Comment (stagraqubole) [inline rubix-presto/pom.xml]: This will cause issues in runtime with engine.

**Comment by pvam** [comment]
Spoke offline. Dependency with other modules is not the beat way to do this. Not yet sure why UT is failing with H2.6 dependency where the yarn method exists.

**Comment by pvam** [comment]
It should be fixed now. Will merge after approval.

**Comment by abhishekdas99** [comment]
Did we test this in presto cluster ? If everything is fine, @pvam can we update this with the result. Then I can approve this .. Also get a approval from @stagraqubole as its Presto related. Lets get this done so that we can go ahead with the new release of Rubix

**Comment by pvam** [comment]
Opened #73, closing this. With #73, we dont dependency on yarn anymore we assume the property is populated via rubix client config.

**Pull request closed without merging**